src/scenes/Wallet/components/EncryptStatusDialog/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Dialog, { DialogTitle, DialogContent, DialogActions } from 'material-ui/Dialog';
+import { Button } from 'material-ui';
+import { withStyles } from 'material-ui/styles';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import _ from 'lodash';
+
+import styles from './styles';
+import appActions from '../../../../redux/App/actions';
+
+@injectIntl
+@withStyles(styles, { withTheme: true })
```

Comment:
if we're not using `classes` here, can we remove this decorator please. Also, don't forget to remove the import and the `styles.js` file and `style` import